Integrations/Securonix/Securonix.py

```diff
@@ -0,0 +1,1196 @@
+    return human_readable, entry_context, watchlist
+
+
+def check_entity_in_watchlist(client: Client, args) -> Tuple[str, Dict, Dict]:
+    """Check if entity is in a watchlist.
+
+    Args:
+        client: Client object with request.
+        args: Usually demisto.args()
+    Returns:
+        Outputs.
+    """
+    entity_id = args.get('entity_id')
+
+    watchlist = client.check_entity_in_watchlist_request(entity_id)  # TODO - real api call since not working in our env
+
+    watchlist_names = watchlist.get('result')
+    if not watchlist_names:
+        human_readable = f'Entity {entity_id} is not a part of any watchlist.'
+        output = {'EntityID': entity_id}
+    else:
+        human_readable = f'Entity {entity_id} is a part of the watchlists: {", ".join(watchlist_names)}.'
+        output = {
+            'EntityID': entity_id,
+            'Watchlistnames': watchlist_names
+        }
+    entry_context = {f'Securonix.EntityInWatchlist(val.EntityID === obj.EntityID)': output}
+    return human_readable, entry_context, watchlist
+
+
+def add_entity_to_watchlist(client: Client, args) -> Tuple[str, Dict, Dict]:
+    """Check if entity is in a watchlist.
+
+    Args:
+        client: Client object with request.
+        args: Usually demisto.args()
+    Returns:
+        Outputs.
+    """
+    watchlist_name = args.get('watchlist_name')
+    entity_type = args.get('entity_type')
+    entity_id = args.get('entity_id')
+    resource_name = args.get('resource_name') if entity_type in ['Resources', 'Activityaccount'] else entity_id
+    expiry_days = args.get('expiry_days') if 'expiry_days' in args else '30'
+
+    watchlist = client.add_entity_to_watchlist_request(watchlist_name, entity_type, entity_id,
+                                                       expiry_days, resource_name)
+
+    if 'Add to watchlist successfull' not in watchlist:
+        raise Exception(f'Failed to add entity {entity_id} to watchlist {watchlist_name}.\n'
+                        f'Error from Securonix is: {watchlist}.')
+    human_readable = f'Added successfully the entity {entity_id} to the watchlist {watchlist_name}.'
+    return human_readable, {}, watchlist
+
+
+def fetch_incidents(client: Client, fetch_time: Optional[str], incident_types: str,
+                    last_run: Dict) -> Tuple[List, Dict]:
+    """Uses to fetch incidents into Demisto
+    Documentation: https://github.com/demisto/content/tree/master/docs/fetching_incidents
+
+    Args:
+        client: Client object with request
+        fetch_time: From when to fetch if first time, e.g. `3 days`
+        incident_types: Incident statuses to fetch, can be: all, open, closed
+        last_run: Last fetch object.
+
+    Returns:
+        incidents, new last_run
+    """
+    timestamp_format = '%Y-%m-%dT%H:%M:%S.%fZ'
+    if not last_run:  # if first time running
+        new_last_run = {'time': parse_date_range(fetch_time, date_format=timestamp_format)[0]}
```

Comment:
are you sure parse date range works and you will always have index [0]?
if not, what will be the error?